src/main/java/frc/trigon/robot/subsystems/shooter/Shooter.java

```diff
@@ -0,0 +1,97 @@
+package frc.trigon.robot.subsystems.shooter;
+
+import com.ctre.phoenix6.controls.VelocityTorqueCurrentFOC;
+import com.ctre.phoenix6.controls.VoltageOut;
+import edu.wpi.first.units.Measure;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.units.Voltage;
+import edu.wpi.first.wpilibj.sysid.SysIdRoutineLog;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import frc.trigon.robot.subsystems.MotorSubsystem;
+import frc.trigon.robot.utilities.ShootingCalculations;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXMotor;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXSignal;
+
+public class Shooter extends MotorSubsystem {
+    private final ShootingCalculations shootingCalculations = ShootingCalculations.getInstance();
+    private final TalonFXMotor
+            rightMotor = ShooterConstants.RIGHT_MOTOR,
+            leftMotor = ShooterConstants.LEFT_MOTOR;
+    private final VelocityTorqueCurrentFOC velocityRequest = new VelocityTorqueCurrentFOC(0);
+    private final VoltageOut voltageRequest = new VoltageOut(0).withEnableFOC(ShooterConstants.FOC_ENABLED);
+    private double
+            targetRightVelocityRotationsPerSecond = 0,
+            targetLeftVelocityRotationsPerSecond = 0;
```

Comment:
convert these variables to local variables inside the reachTargetShootingVelocityFromShootingCalculations method